src/pages/projects/AddProjectContributorModal.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from 'react-i18next';
+import { Modal } from '../../components/Modal';
+import { AddProjectContributorForm } from './AddProjectContributorForm';
+
+interface AddProjectContributorModalProps {
+  open: boolean;
+  hasProjectManager: boolean;
+  toggleModal: () => void;
+}
+
+export const AddProjectContributorModal = ({
```

Comment:
skjer veldig lite  i komponenten her, så det er et spørsmål om det er verdt en egen fil man må navigere seg gjennom når den bare setter noen props. Ser den bare brukes en plass per nå også.

Reply:
Enig

Reply:
Jeg prøvde dette, men jeg syns det ble mye mer rot å lete blant modal-koden for å finne skjema-koden, så revertet.